test/built-ins/Temporal/now/plainDateTime/calendar-function.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2020 Igalia, S.L. All rights reserved.
```

Comment:
Is this correct?

Reply:
Yes, originates here: https://github.com/tc39/proposal-temporal/commit/60211b2c99b6169665c4f5e0bec8a5a34a77940b